frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -486,6 +515,7 @@ export function remove(url: string, requestOptions = {}) {
 export async function streamResult(
   url: string,
   name: string,
+  queryParams: QueryParameters | undefined,
```

Comment:
The `streamResult` function is exported, which means this breaks API for plugins.
Since the `queryParams` can be undefined, we can easily avoid braking API by adding as optional, to the end of the params. i.e `queryParams?: QueryParamters` after `errCb`.

Reply:
Make sense!